### ISSUE TITLE
Show graph data for additional categories

### ISSUE THREAD

**Issue by tlenhart** [tlenhart/ProvoDashboard#8]
Currently data is only being show for data in the transportation gallery. 

Data needs to be shown for each of the five additional categories.

**Comment by tlenhart** [comment]
Update: Right now it is only safety (and safety again on index) that are showing graphs. Should be easy enough to change, some code just needs to be copied to the other pages and one option (an item passed to a function) changed.

**Comment by EffCeePee** [comment]
Do you want the graph type for the categories to be left blank or to say
base?

**Comment by tlenhart** [comment]
I don't think it would make too much of a difference either way. Personally I would probably prefer the default type to be base instead of blank.

Are you talking about in models.py or on the html pages. (Or somewhere else?)

**Comment by EffCeePee** [comment]
I'm talking as the type in the database.

**Comment by tlenhart** [comment]
Hmmm. Let's do base... Actually, maybe "none" would be a better option? That way it would be clear to whomever is editing/using it that a graph type is not selected.

I think we might also want to change GRAPHCHOICE from: 
GRAPHCHOICE = (
    ("chart-polar", "Polar"),
    ("chart-bar", "Bar"),
    ("chart-line", "Line"),
    ("chart-base", "Base"),
    ("chart-doughnut", "Doughnut"),
    ("chart-pie", "Pie"),
    ("chart-radar", "Radar")
)
to: 
GRAPHCHOICE = (
    ("none", "None"),
    ("polar", "Polar"),
    ("bar", "Bar"),
    ("line", "Line"),
    ("doughnut", "Doughnut"),
    ("pie", "Pie"),
    ("radar", "Radar")
)

The way the graph type to render is picked in the js code is the first element of the tuple.
Now that I've thought about it, I think none would be the best option.

**Comment by EffCeePee** [comment]
Ok, if we pick None which graph type shows?  Would it be better to do
default?

**Comment by tlenhart** [comment]
There isn't really a default graph type. Base is just something used by chart.js to signify a graph that will have its type specified. If the type of graph is none we *could* just pick a type to display the data as, but stuff might not display in the best way if we go this route.

Picking none wouldn't show anything, but what I can do is place a message in the div that contains the graph that states an invalid graph type was set. (When the graph type is none or invalid.) That way whoever is entering the data would know that they need to go back in and change the graph type.

**Comment by EffCeePee** [comment]
I think that would be best, that will help with our testing and error
checking as well.

Ok, I'll get this done and send it over with graph type none.
